great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -0,0 +1,212 @@
+           4.4: Insure that the metric_value is numeric (ranges can be computed for numeric-valued metrics only).
+           4.5: Append the value of the computed metric to the list (one for each batch_id -- loop iteration variable).
+        5. Convert the list of floating point metric computation results to a numpy array (for further computations).
+        6. Compute the mean and the standard deviation of the metric (aggregated over all the gathered Batch objects).
+        7. Compute the number of standard deviations (as a floating point number rounded to the nearest highest integer)
+           needed to create the "band" around the mean so as to achieve the specified false_positive_rate (note that the
+           false_positive_rate of 0.0 would result in an infinite number of standard deviations, hence it is "nudged" by
+           a small quantity, "epsilon", above 0.0 if false_positive_rate of 0.0 is provided as argument in constructor).
+           (Please refer to "https://en.wikipedia.org/wiki/Normal_distribution" and references therein for background.)
+        8. Compute the "band" around the mean as the min_value and max_value (to be used in ExpectationConfiguration).
+        9. Set up the arguments for and call build_parameter_container() to store the parameter as part of "rule state".
+        """
+
+        batch_ids: List[str] = self.get_batch_ids(batch_ids=batch_ids)
+        if not batch_ids:
+            raise ge_exceptions.ProfilerExecutionError(
+                message=f"Utilizing a {self.__class__.__name__} requires a non-empty list of batch identifiers."
+            )
+
+        # Obtaining domain kwargs from rule state (i.e., variables and parameters); from instance variable otherwise.
+        if isinstance(
+            self._metric_domain_kwargs, str
+        ) and self._metric_domain_kwargs.startswith("$"):
+            metric_domain_kwargs = get_parameter_value(
+                fully_qualified_parameter_name=self._metric_domain_kwargs,
+                domain=domain,
+                variables=variables,
+                parameters=parameters,
+            )
+        else:
+            metric_domain_kwargs = self._metric_domain_kwargs
+
+        # Obtaining value kwargs from rule state (i.e., variables and parameters); from instance variable otherwise.
+        if (
+            self._metric_value_kwargs is not None
+            and isinstance(self._metric_value_kwargs, str)
+            and self._metric_value_kwargs.startswith("$")
+        ):
+            metric_value_kwargs = get_parameter_value(
+                fully_qualified_parameter_name=self._metric_value_kwargs,
+                domain=domain,
+                variables=variables,
+                parameters=parameters,
+            )
+        else:
+            metric_value_kwargs = self._metric_value_kwargs
+
+        expectation_suite_name: str = f"tmp_suite_domain_{domain.id}"
```

Comment:
This is not saved so should not have collisions, but maybe add a random string to ensure (e.g. from our SQL temp table logic)

Reply:
Done -- in next commit.